packages/frontend-api/src/Model/Mutation/Payment/Exception/MaxTransactionCountReachedUserError.php

```
@@ -2,14 +2,14 @@
 
 declare(strict_types=1);
 
-namespace App\FrontendApi\Mutation\Payment\Exception;
+namespace Shopsys\FrontendApiBundle\Model\Mutation\Payment\Exception;
 
 use Overblog\GraphQLBundle\Error\UserError;
 use Shopsys\FrontendApiBundle\Model\Error\UserErrorWithCodeInterface;
 
 class MaxTransactionCountReachedUserError extends UserError implements UserErrorWithCodeInterface
 {
-    private const CODE = 'max-transaction-count-reached';
+    public const CODE = 'max-transaction-count-reached';
```

I'm not sure about using the constant in the tests. 
Changing the value is not a safe operation – Storefront takes into account the value. So I think it is fair to let the test fail when the value is changed

---

Hmm, I have been doing it like this for a while now :smile: You have a valid point, however - if the test fails after they change the code, will they know they should fix it in the storefront file, or will they just adjust the value in the test? :thinking:

---

changed in 9de72b2fdc